### ISSUE TITLE
Cannot be found as a Matrix client in F-Droid

### ISSUE THREAD

**Issue by djmaze** [krille-chan/fluffychat#895]
### Bug Description

When search for a matrix client in F-Droid, I do not find Fluffychat.

### Steps to Reproduce

1. Go to search in the F-Droid android app.
2. Search for "matrix" or "matrix cleint"

### Expected Behavior

As a Matrix newbie, I'd expect to find FluffyChat in the results there.

### App Version

1.17.3

### Additional Platform Information

Android

**Comment by aCursedComrade** [comment]
This is more of a shortcoming of F-Droid android client. It looks only at the application name and its sub-heading during the search. The web version at https://search.f-droid.org/ does not have this problem. There are discussions related to this in their forum but https://forum.f-droid.org/ is down at the moment.

**Comment by licaon-kter** [comment]
yup known search issues :crying_cat_face:

**Comment by BhasherBEL** [comment]
> This is more of a shortcoming of F-Droid android client. It looks only at the application name and its sub-heading during the search. The web version at https://search.f-droid.org/ does not have this problem. There are discussions related to this in their forum but https://forum.f-droid.org/ is down at the moment.

Indeed, but the description could be adapted to make it easier to find, to something like "Chat with your friends with FluffyChat trough Matrix"
